tables/de-de-comp8.ctb

```diff
@@ -29,8 +29,6 @@
 #
 #+locale: de
 #+type: computer
-#+contraction: no
```

Comment:
Are you saying that type computer implies no contraction and grade 0?

Reply:
> type computer implies no contraction

Yes.

> and grade 0

Not really. Actually grade 0 is reserved for Basisschrift. When both that table and the computer braille table are tagged with grade 0 this can result in the wrong table being loaded for a grade 0 query. This was actually happening in Pipeline.

Basically grade and contraction just have no business being present in a computer braille table.